samples/bot-task-module/csharp/Pages/PowerApp.cshtml

```diff
@@ -0,0 +1,40 @@
+﻿@page
+@using Microsoft.Extensions.Configuration
+@inject IConfiguration Configuration
+@{
+    ViewData["Title"] = "PowerApp";
+    Layout = "~/Pages/Shared/_Layout.cshtml";
+}
+
+<iframe width="720" height="520" style="width: 94%;" src="https://web.powerapps.com/webplayer/iframeapp?source=iframe&amp;screenColor=rgba(104,101,171,1)&amp;appId=/providers/Microsoft.PowerApps/apps/f998c01d-522c-46ad-9690-f67f271afdfe"></iframe>
```

Comment:
Do we need this id?

Reply:
Add a place holder like <PowerBI-App-Id> and explain it in Readme for update and how to get this id?

Reply:
Comment Solved